client/client.go

```diff
@@ -38,8 +38,8 @@ type DataExportClient interface {
 
 // Client represents a HTTP client for making requests to the FullStory API.
 type Client struct {
-	HTTPClient *http.Client
-	Config     *config.Config
+	HTTPClient    *http.Client
+	Config *config.Config
```

Comment:
These changes are unnecessary.

Reply:
Can you remove these changes? This will contribute to `checkgofmt` errors.

Reply:
Believe that gofmt was needed to get rid of the spaces there.